### PR TITLE
Move alexandria to sopthiatxd

### PR DESCRIPTION
**Pull request by JakubFornadel**

### CODE REVIEW

**Comment by ejossev** [comment]
Can libraries/alexandria be removed?

**Comment by JakubFornadel** [comment]
> Can libraries/alexandria be removed?

Yes, after SDK's start to use alexandria_plugin,  libraries/alexandria is useless.

File: libraries/plugins/apis/alexandria_api/alexandria_api_asset.cpp [hunk 1]
```diff
@@ -0,0 +1,75 @@
+#include <sophiatx/plugins/alexandria_api/alexandria_api_asset.hpp>
+
+
+namespace {
+
+int64_t precision( const sophiatx::protocol::asset_symbol_type& symbol )
```
Comment (MatusKysel) [inline libraries/plugins/apis/alexandria_api/alexandria_api_asset.cpp]: Potrebujeme vobec api_asset? nestaci to riesit ako string?

File: libraries/plugins/apis/alexandria_api/alexandria_api_impl.cpp [hunk 2]
```diff
@@ -0,0 +1,1378 @@
...
+
+      if( start.witnesses.size() == 0 )
+         return result;
+
+      start_key.push_back( fc::variant( start.witnesses[0].votes ) );
+      start_key.push_back( fc::variant( start.witnesses[0].owner ) );
+   }
+
+   auto limit = args.limit;
+   auto witnesses = _database_api->list_witnesses( { fc::variant( start_key ), limit, database_api::by_vote_name_reverse } ).witnesses;
```
Comment (MatusKysel) [inline libraries/plugins/apis/alexandria_api/alexandria_api_impl.cpp]: toto zmen prosimta podla tohoto fixu https://github.com/SophiaTX/SophiaTX/pull/126/files

File: libraries/plugins/apis/alexandria_api/alexandria_api_impl.cpp [hunk 3]
```diff
@@ -0,0 +1,1378 @@
...
+   op.sponsor = args.sponsoring_account;
+   op.sponsored = args.sponsored_account;
+   op.is_sponsoring = args.is_sponsoring;
+
+   result.op = std::move(op);
+   return result;
+}
+
+} } } // sophiatx::plugins::alexandria_api
+
```
Comment (MatusKysel) [inline libraries/plugins/apis/alexandria_api/alexandria_api_impl.cpp]: toto asi tiez mozes vymazat, je to blbost ale je to zbytocne

File: libraries/plugins/apis/alexandria_api/alexandria_api_impl.cpp [hunk 4]
```diff
@@ -0,0 +1,1378 @@
...
```
Comment (MatusKysel) [inline libraries/plugins/apis/alexandria_api/alexandria_api_impl.cpp]: Este jedna poznamka vseobecne k tejto api, v povodnej verzii malo vela volani try - FC_CAPTURE_AND_RETHROW
tu to netreba?

**Pull request merged**